saw-core/src/Verifier/SAW/SharedTerm.hs

```diff
@@ -1077,62 +1084,88 @@ scDefTerm sc d = scGlobalDef sc (defIdent d)
 scApplyCtor :: SharedContext -> Ctor -> [Term] -> IO Term
 scApplyCtor sc c args = scCtorApp sc (ctorName c) args
 
+-- | Create a term from a sort.
 scSort :: SharedContext -> Sort -> IO Term
 scSort sc s = scFlatTermF sc (Sort s)
 
+-- | Create a literal term from a natural number.
 scNat :: SharedContext -> Natural -> IO Term
 scNat sc n = scFlatTermF sc (NatLit n)
 
+-- | Create a literal term from a string.
 scString :: SharedContext -> String -> IO Term
 scString sc s = scFlatTermF sc (StringLit s)
 
+-- | Create a term representing the global type of strings.
```

Comment:
Might be more precise to say "the primitive saw-core type `String`" (i.e. `@String@` in haddock syntax)